### ISSUE TITLE
Possible typo in the `sar` metafile

### ISSUE THREAD

**Issue by balandongiv** [open-mmlab/mmocr#1115]
The [mmocr tutorial](https://github.com/open-mmlab/mmocr/blob/main/demo/MMOCR_Tutorial.ipynb)  demonstrated how to perform inference for SAR on colab.

In one section, the tutorial showed the reason to change the learning rate
```
# The original learning rate (LR) is set for 8-GPU training.
# We divide it by 8 since we only use one GPU.
cfg.optimizer.lr = 0.001 / 8
```

However, the `sar` [metafile](https://github.com/open-mmlab/mmocr/blob/dab5725d70b92ada4383565c3e60f6634616fd60/configs/textrecog/sar/metafile.yml#L7) indicate the predefined `0.001`  was obtained based on experimentation with 48 GPUs.

However, the textsnake [metafile](https://github.com/open-mmlab/mmocr/blob/dab5725d70b92ada4383565c3e60f6634616fd60/configs/textdet/textsnake/metafile.yml#L7) indicate the team was using a Training Resources: 8x GeForce GTX 1080 Ti.

I am not sure whether the `48 GPUs` is typo or something else?

**Comment by xinke-wang** [comment]
Hi, by checking the training log of SAR [here](https://download.openmmlab.com/mmocr/textrecog/sar/20210330_105728.log.json),
`\ngpu_ids = range(0, 48)` indicates the model was trained on 48 GPUs. However, our recent experiments show that these models can be trained within much less GPU resources. New pre-trained models will be released next month.

**Comment by balandongiv** [comment]
Seem there is misunderstanding in the [mmocr tutorial](https://github.com/open-mmlab/mmocr/blob/main/demo/MMOCR_Tutorial.ipynb).

In that case, the `lr` should be

`cfg.optimizer.lr = 0.001 / 48`

**Comment by xinke-wang** [comment]
That might be a typo in the tutorial.

**Comment by gaotongxiao** [comment]
It could be a typo, but dividing the learning rate by 48 might incur a long training process for just a demo task.
